plugin.xml

```diff
@@ -70,7 +70,7 @@ SOFTWARE.
             </intent-filter>
         </config-file>
         <source-file src="src/android/io/branch/BranchSDK.java" target-dir="src/io/branch" />
-        <framework src="io.branch.sdk.android:library:1.+" />
+        <framework src="io.branch.sdk.android:library:1.+" custom="true" type="gradleReference" />
```

Comment:
Sorry, I meant to show an example `framework` tag with the `"com.android.support:appcompat-v7:21+"` dependency as shown in plugin reference guide. 

For Branch sdk we only need to add `<framework src="io.branch.sdk.android:library:1.+" />` so previous code was fine (it should work) and doesn't need any attributes i.e custom or type.